aries-site/src/examples/components/checkbox/CheckBoxDescriptionExample.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, Form, FormField } from 'grommet';
+
+export const CheckBoxDescriptionExample = () => {
+  const [checked, setChecked] = useState();
+
+  return (
+    <Box width="medium" align="center">
+      <Form>
+        <FormField
+          name="checkbox-with-desc"
+          htmlFor="checkbox-desc"
+          label="Checkbox Heading"
+          help="Checkbox Description"
+        >
+          <CheckBox
+            name="checkbox-with-desc"
+            id="checkbox-desc"
+            label="Checkbox Label"
+            checked={checked}
+            onChange={event => setChecked(event.target.checked)}
+          />
+        </FormField>
+      </Form>
+    </Box>
```

Comment:
Remove unnecessary Box 

```suggestion
      <Form>
        <FormField
          name="checkbox-with-desc"
          htmlFor="checkbox-desc"
          label="Checkbox Heading"
          help="Checkbox Description"
        >
          <CheckBox
            name="checkbox-with-desc"
            id="checkbox-desc"
            label="Checkbox Label"
            checked={checked}
            onChange={event => setChecked(event.target.checked)}
          />
        </FormField>
      </Form>
```